package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "leaderboard",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "build": "webpack"
```

Comment:
- [ ] Kindly make sure you have all the needed scripts to run the project locally with the webpack bundler.
`HInt: YOu may consider having these script`
```json
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack --config webpack.config.js --watch"
```

Reply:
Hi @christianbiring1, thank you for the review